libtonode-tests/tests/sync.rs

```diff
@@ -31,7 +31,7 @@ async fn sync_mainnet_test() {
     let mut lightclient = LightClient::create_from_wallet_base_async(
         WalletBase::from_string(HOSPITAL_MUSEUM_SEED.to_string()),
         &config,
-        2_611_700,
+        2_715_150,
```

Comment:
If this is, for example a NU height, or protocol epoch, I think it would be best to import the most general relevant constant and us it explicitly.

If it's an HMS-specific birthday, then maybe  we should associate it with that seed in some ergomic way, so that e.g. other tests can get the seed-associated bdday? 

If it's not, for example if it's a birthday that's idiosyncratic to this test, then let's throw some kind of comment on it, specifying that.

Reply:
This test will be removed eventually it is ignored and I only use it sometimes when a larger chain is needed. This birthday is just a block height near to the server tip at time of writing.